doc/source/working_with_kiwi/runtime_configuration_incl.rst

```diff
@@ -0,0 +1,99 @@
+.. This file contains the documentation of the runtime configuration, but
+   it is not directly included into the toctree, because it needs to be
+   included in two places with only one label.
+   => In one location it is included with a label preceeding it and in the
+      other (the man page) without the label.
+
+The Runtime Configuration File
+==============================
+
+KIWI supports an additional configuration file for runtime specific
+settings that do not belong into the image description but which are
+persistent and would be unsuitable for command line parameters.
+
+The runtime configuration file must adhere to the `YAML
+<https://yaml.org/>`_ syntax. KIWI searches for the runtime configuration
+file in the following locations:
+
+1. :file:`~/.config/kiwi/config.yml`
+
+2. :file:`/etc/kiwi.yml`
+
+
+The parameters that can be tweaked via the runtime configuration file are
+illustrated in the following example:
+
+.. code-block:: yaml
+
+   xz:
+     # Additional command line flags for `xz`
+     # see `man xz` for details
+     - options: -9e
+
+   obs:
+     # Override the URL to the Open Build Service (i.e. how repository
+     # paths starting with `obs://` will be resolved).
+     # This setting is useful for building a KIWI appliance, which is
```

Comment:
Move "locally" a bit further to avoid nested sentences.

```suggestion
     # This setting is useful for building a KIWI appliance locally, which is
```